APCAppCore/APCAppCore/UI/Components/APCBadgeLabel.m

```diff
@@ -92,4 +92,11 @@ - (void)setText:(NSString *)text
     [self setNeedsDisplay];
 }
 
+- (void) setTintColor:(UIColor *)tintColor
+{
+    _tintColor = tintColor;
+    self.textColor = tintColor;
+    self.layer.borderColor  = self.tintColor.CGColor;
+}
```

Comment:
AppCore might be messed up but should implement `-tintColorDidChange:` instead. Could you please verify that doing so won't work and add a comment to that effect if that's the case? Thanks.

Just checked the source code. Okay, If you could please fix this, that would be great.

1. Need to remove the tintColor property from the header.
2. Replace `_tintColor = [UIColor appPrimaryColor];` with `self.tintColor = [UIColor appPrimaryColor];` in `sharedInit`
3. Replace the method signature for your method with `- (void) tintColorDidChange` and use self.tintColor

Thanks.